### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by hzhou**
:wave:! GitHub Classroom created this pull request as a place for your teacher to leave feedback on your work. It will update automatically. **Don’t close or merge this pull request**, unless you’re instructed to do so by your teacher.

In this pull request, your teacher can leave comments and feedback on your code. Click the **Subscribe** button to be notified if that happens.

Click the **Files changed** or **Commits** tab to see all of the changes pushed to `master` since the assignment started. Your teacher can see this too.

<details>
<summary>
<strong>Notes for teachers</strong>
</summary>

Use this PR to leave feedback. Here are some tips:

- Click the **Files changed** tab to see all of the changes pushed to `master` since the assignment started. To leave comments on specific lines of code, put your cursor over a line of code and click the blue **+** (plus sign). To learn more about comments, read “[Commenting on a pull request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)”.
- Click the **Commits** tab to see the commits pushed to `master`. Click a commit to see specific changes.
- If you turned on autograding, then click the **Checks** tab to see the results.
- This page is an overview. It shows commits, line comments, and general comments. You can leave a general comment below.

For more information about this pull request, read “[Leaving assignment feedback in GitHub](/help/leaving-feedback-in-github)”.
</details>

### CODE REVIEW

File: billboard_1801_bronze.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
+/*
+Bill Zheng
+Time: 26 min
+USACO Test Cases: **********
+1-10 Difficulty: 2-3, how to do it was really simple
+Reflection: Only difficulty was checking edge cases and looking back at my code. After first attempt I had 8/10 of the test cases passed
+	and I had to go back and look through my code to find my mistakes which took another 10 min or so.
```
Comment (hzhou) [inline billboard_1801_bronze.java:7]: good.

File: billboard_1801_bronze.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+//--------------------------------------------------------------------------------------------------------------------------
+		int ax1 = scan.nextInt();
+		int ay1 = scan.nextInt();
+		int ax2 = scan.nextInt();
+		int ay2 = scan.nextInt();
+		
+		int bx1 = scan.nextInt();
+		int by1 = scan.nextInt();
+		int bx2 = scan.nextInt();
+		int by2 = scan.nextInt();
```
Comment (hzhou) [inline billboard_1801_bronze.java:26]: ✔️

File: billboard_1801_bronze.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+//--------------------------------------------------------------------------------------------------------------------------
+		if (by2 >= ay2 && by1 <= ay1) {
+			if (bx1 < ax2 && bx1 > ax1 && bx2 >= ax2) {
+				area = (ay2-ay1)*(bx1-ax1);
+			}
+			if (bx2 < ax2 && bx2 > ax1 && bx1 <= ax1) {
+				area = (ay2-ay1)*(ax2-bx2);
+			}
+		}
+//--------------------------------------------------------------------------------------------------------------------------
```
Comment (hzhou) [inline billboard_1801_bronze.java:46]: This is OK. You could help yourself reading the code simply by adding a short comment above each block.

File: lifeguards_1801_bronze.java [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+		Scanner scan = new Scanner (new File ("lifeguards.in"));
+//--------------------------------------------------------------------------------------------------------------------------
+		int N = scan.nextInt();
+		int[][] shifts = new int[N][2]; //inputs lifeguards shifts into a 2D array/matrix
+		for (int i = 0; i < N; i++) {
+			shifts[i][0] = scan.nextInt();
+			shifts[i][1] = scan.nextInt();
+		}
+//--------------------------------------------------------------------------------------------------------------------------
+		int[] time = new int[1001]; //timeline array to track hours
```
Comment (hzhou) [inline lifeguards_1801_bronze.java:24]: This is a USACO *bronze* only hacky solution. It does not scale to higher levels or practical problems. Try work out alternate solution.

File: outofplace_1801_bronze.java [hunk 5]
```diff
@@ -0,0 +1,43 @@
+/*
+Bill Zheng
+Time: 36 min
+USACO Test Cases: **********
+1-10 Difficulty: 6, need to find the right solution
+Reflection: Initially I forgot how to do this problem and so I was trying to work a super long method of swapping cows and checking conditions
+	to see if the cow was in the right place. I did this for about 30 min until I realized it would be way to hard to check all of the edge
+	cases. And so, after 30 min I decided to look back at some code I did in the past and I found the easiest method to solve the problem
+	by comparing two arrays, one sorted and one not sorted. The code for the final solution took like 2 minutes to write. This entire problem
+	was just me digging into a dead end and then looking back to find the correct and easiest solution.
```
Comment (hzhou) [inline outofplace_1801_bronze.java:10]: Clever solution is nice, but it is also a win-it-all/lose-it-all deal. You need practice on logics that your *intuition* can relate to so you can reliably produce results.
